VideoLocker/build.gradle

```diff
@@ -84,20 +84,28 @@ dependencies {
     }
     compile fileTree(dir: 'libs', include: '*.jar')
     compile 'com.facebook.android:facebook-android-sdk:3.22.+'
-    compile 'com.google.android.gms:play-services:6.5.+'
+    compile 'com.google.android.gms:play-services:7.5.0'
     compile 'com.google.code.gson:gson:2.2.4'
     compile 'de.greenrobot:eventbus:2.4.0'
     compile 'com.squareup.phrase:phrase:1.0.3'
     compile 'com.squareup.okhttp:okhttp:2.5.0'
     compile 'com.squareup.okhttp:okhttp-urlconnection:2.5.0'
     compile 'com.squareup.retrofit:retrofit:1.9.0'
-    compile 'com.android.support:support-v4:21.1.0'
+    compile 'com.android.support:support-v4:22.2.0'
```

Comment:
appcompat 22 is designed to target API level 22. Can you also update `targetSdkVersion` to 22?